doc/cephfs/upgrading.rst

```diff
@@ -39,37 +37,20 @@ command. Older versions of Ceph require you to stop these daemons manually.
 
     ceph status # wait for MDS to finish stopping
 
-4. Take all standbys offline, e.g. using systemctl:
-
-::
-
-    systemctl stop ceph-mds.target
-
-5. Confirm only one MDS is online and is rank 0 for your FS:
-
-::
-
-    ceph status
-
-6. Upgrade the single active MDS, e.g. using systemctl:
-
-::
-
-    # use package manager to update cluster
-    systemctl restart ceph-mds.target
-
-7. Upgrade/start the standby daemons.
+4. For each MDS, upgrade packages and restart. Note: to reduce failovers, it is
+   recommended -- but not strictly necessary -- to first upgrade standby daemons.
```

Comment:
I think this needs to be louder and clearer, so that people don't leave old versions of the code running without realizing that it won't help them. As is it just reads like "yeah you should upgrade the other daemons for a better experience".

Reply:
I'm not sure we're on the same page. This note is saying that they should upgrade standbys **first**. Of course, they should upgrade all the daemons. So I'm not sure what you mean "leave old versions of the code running".